plugins/pulp_rpm/plugins/migrations/0024_errata_pushcount_string.py

```diff
@@ -0,0 +1,33 @@
+"""
+This migration casts the pushcount field as a String.
+
+For the units_erratum collection, any object with the field 'pushcount' storing a non-String is
+cast to a String and saved. This is required since non String values will not validate with the
+Mongoengine definition.
+"""
+from pulp.server.db import connection
+
+
+def migrate(*args, **kwargs):
+    """
+    Perform the migration as described in this module's docblock.
+
+    :param args:   unused
+    :type  args:   list
+    :param kwargs: unused
+    :type  kwargs: dict
+    """
+    db = connection.get_database()
+    errata_collection = db['units_erratum']
+
+    for erratum in errata_collection.find():
```

Comment:
If you think not all documents have this field, you could improve performance by doing:

```
for erratum in errata_collection.find({'$exists': {'pushcount': true}}):
```

That could make a big difference for users with large data sets.

You can also limit the fields returned (and thus how much data goes over the wire) by using the second argument to find, which I think would look like this;

```
for erratum in errata_collection.find({'$exists': {'pushcount': true}}, {'pushcount': 1}):
```